tests/bench-paper.rkt

```diff
@@ -17,7 +17,7 @@
 
 ;; Staging the evalo relation running on (eval <program-text>) (thus faster eval functions in particular)
 (require "applications/or-lang-interp.rkt")
-(require "applications/proof.rkt")
+(require (submod "applications/proof.rkt" benchmark))
```

Comment:
ah, ok

Reply:
Right. In order to make sure we run the tests in that file, I need to be calling the submod from the benchmark script, which I wasn't doing before; part of the fix for the previous.